src/LuaDebuggerLauncher/tools/CommonProjectSystemExtension.targets

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should have come from a Nuget package as well. Did you install  the `microsoft.visualstudio.projectsystem.sdk.tools` package?